### DISCUSSION TITLE
[Unstable] Stackable items drop from Fabricator

### DISCUSSION THREAD

**Discussion by FlashORC** [FakeFishGames/Barotrauma#14847] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Some stackable items drop from Fabricator while crafting multiple instances. Noticed on Fuel Rods, Batteries, Fuel and Oxygen tanks with 100% chance of Excellent or Masterwork quality.
Not reproducible with revolver rounds, shotgun shells, etc.
Both outpost and ship Fabricators.
No errors in console.

### Reproduction steps

1. Start fabrication of 8 Fuel Rods with 100% chance of any quality.
2. First one stays in output slot
3. Other 7 drop on the floor

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.6.4.0 (unstable)

### -

v1.6.16.0

### Which operating system did you encounter this bug on?

Windows

**Comment by pavel4win** [comment]
The same thing happened to me in a network game.

Unzip the attached save.
Enter as the character "paula" (edit steam id in lonelyWolf20241018_CharacterData) 
Create a server, load a save, find galena in the closet.
Dismantle galena and order the crafting of 2 uranium fuel rods. The rods will be blue since paula has 100 mechanics.
When the first rod is crafted, it will remain in the output of the fabricator.
When the second rod is crafted (the same blue color), it will fall out on the floor.
Tested on c4 also.

[1.zip](https://github.com/user-attachments/files/17492053/1.zip)

**Comment by Regalis11** [comment]
This has now been fixed in out private repo, fix coming up in the next hotfix.
